src/ol/render/canvas.js

```diff
@@ -389,6 +388,7 @@ export function getTextDimensions(baseStyle, chunks) {
     lineWidth += currentWidth;
     const currentHeight = measureTextHeight(font);
     heights.push(currentHeight);
+    height += currentHeight;
     lineHeight = Math.max(lineHeight, currentHeight);
```

Comment:
```suggestion
```

And the `lineHeight` variable defined in line 376 is no longer needed.

Reply:
Yes，you are right.